superbench/benchmarks/micro_benchmarks/cublas_para_info.json

```diff
@@ -0,0 +1,175 @@
+[
```

Comment:
@abuccts how can we handle it?

Reply:
convert json file to built-in dict list in init of the benchmark，and add a interface for user to run custom configuration with json/yaml str